.github/workflows/bump_dependencies.yml

```diff
@@ -9,12 +9,23 @@ jobs:
     if: startsWith(github.repository, 'spinnaker/')
     runs-on: ubuntu-latest
     steps:
+    # Halyard releases are separate from Spinnaker releases so we need to
+    # exclude Halyard from bump deps when targeting services release-* branches.
+    - name: decide bumpdep target repositories
+      id: bumpdep_targets
+      run: |
+        REPOS=clouddriver,echo,front50,gate,halyard,igor,keel,orca
+        if [ "${{ github.event.client_payload.branch }}" != "  origin/master"]; then
```

Comment:
Based on [this GHA run](https://github.com/spinnaker/fiat/runs/5951468606?check_suite_focus=true):
```
Run BRANCHES=$(git branch -r --contains refs/tags/v1.31.0)
BRANCHES is '  origin/master
  origin/release-1.28.x'
NUM_BRANCHES is '2'
exactly one branch required to release fiat, but there are 2 (  origin/master
  origin/release-1.28.x)
```